src/components/CytoscapeGraph/ContextMenu/NodeContextMenu.tsx

```diff
@@ -87,8 +90,8 @@ export class NodeContextMenu extends React.PureComponent<Props> {
   }
 
   render() {
-    // Disable context menu if we are dealing with a unknown or an inaccessible node
-    if (this.props.nodeType === NodeType.UNKNOWN || this.props.isInaccessible) {
+    // Disable context menu if we are dealing with an aggregate (currently has no detail) or an inaccessible node
+    if (this.props.nodeType === NodeType.AGGREGATE || this.props.isInaccessible) {
```

Comment:
Are `unknown` nodes inaccessible? If yes, I can understand why you removed the `this.props.nodeType === NodeType.UNKNOWN` test from the condition. If no, hmm... then I don't understand :)

Reply:
Yes, unknown nodes are always inaccessible.